test/extended/cli/observe.go

```diff
@@ -107,4 +103,10 @@ var _ = g.Describe("[sig-cli] oc observe", func() {
 		o.Expect(out).To(o.ContainSubstring("Sync should_be_passed"))
 		o.Expect(os.Unsetenv("MYENV")).NotTo(o.HaveOccurred())
 	})
+
+	g.It("works as expected with cluster operators [apigroup:config.openshift.io]", func() {
```

Comment:
Why separate when the original `works as expected [apigroup:config.openshift.io]` name still keeps the apigroup?

Reply:
Done